### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

File: docs/edgeql/operators/bytes.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+=====
+
+Much like with :ref:`arrays <ref_eql_expr_array_elref>` individual bytes
+and bytes slices can be produced by using ``[]``:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT b'some bytes'[1];
+    {b'o'}
+    db> SELECT b'some bytes'[1:3];
```
Comment (1st1) [inline docs/edgeql/operators/bytes.rst]: Please add a negative slice example.

File: docs/edgeql/functions/converters.rst [hunk 2]
```diff
@@ -8,6 +8,216 @@ These functions convert between different scalar types. When a
...
+                    day: int64, hour: int64, min: int64, sec: float64) \
+                    -> naive_datetime
+
+    :index: parse naive_datetime
+
+    Create a :eql:type:`naive_datetime` value.
+
+    Similar to :eql:func:`to_datetime`, the :eql:type:`naive_datetime`
+    value can be parsed from the input :eql:type:`str` *s* with an
+    optional *fmt* argument or it can be given in terms of it's
```
Comment (1st1) [inline docs/edgeql/functions/converters.rst]: it's -> its

File: docs/edgeql/functions/converters.rst [hunk 3]
```diff
@@ -8,6 +8,216 @@ These functions convert between different scalar types. When a
...
+
+
+.. eql:function:: std::to_float32(s: str, fmt: OPTIONAL str={}) -> float32
+
+    :index: parse float32
+
+    Create a :eql:type:`float32` value.
+
+    Parse a :eql:type:`float32` from the input *s* and optional format
+    specification *fmt*.
```
Comment (1st1) [inline docs/edgeql/functions/converters.rst]: Can we copy/paste Postgres format specs to our docs?  See `tx_start.rst` on how to deal with copyright/licensing.

File: docs/edgeql/operators/bytes.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/edgeql/operators/bytes.rst]: Added.

File: docs/edgeql/functions/converters.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/edgeql/functions/converters.rst]: Fixed all instances of this.

File: docs/edgeql/functions/converters.rst (hunk 3 above)
Comment (vpetrovykh) [reply docs/edgeql/functions/converters.rst]: Added format specs.

File: docs/edgeql/functions/converters.rst [hunk 4]
```diff
@@ -49,3 +385,251 @@ the functions below allow more options for such conversions.
             "a": 2,
             "b": "hello"
         }'}
+
+
+Formatting
+----------
```
Comment (1st1) [inline docs/edgeql/functions/converters.rst]: This is a top level section, and "Date and time formatting options" is second level?  If so, this should have different underline style.

File: docs/edgeql/functions/converters.rst [hunk 5]
```diff
@@ -49,3 +385,251 @@ the functions below allow more options for such conversions.
...
+the *FX* prefix modifier is used. For example:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT to_naive_date(
+    ...     '2000    JUN', 'YYYY MON');
+    {<naive_date>'2000-06-01'}
+    db> SELECT to_naive_date(
+    ...     '2000    JUN', 'FXYYYY MON');
+    InternalServerError: invalid value "   " for "MON"
```
Comment (1st1) [inline docs/edgeql/functions/converters.rst]: Ideally we should fix `errmech.py` to recognize Postgres' error and convert it to a proper EdgeDB error.  Please add a new issue for that.

File: docs/edgeql/functions/converters.rst (hunk 4 above)
Comment (vpetrovykh) [reply docs/edgeql/functions/converters.rst]: I've changed the nesting level for the sections.

**Pull request merged**